pkg/fileutils/fileutils_test.go

```diff
@@ -384,73 +376,82 @@ func TestMatches(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		res, _ := regexpMatch(test.pattern, test.text)
+		pm, err := NewPatternMatcher([]string{test.pattern})
+		if err != nil {
```

Comment:
Do you have a test for an error case where err is supposed to be set? Just to make sure an error in a pattern really is caught and returned? I looked but I didn't see one.